docs/pai-protocol.yaml

```diff
@@ -0,0 +1,218 @@
+protocolVersion: String, required # Protocol version, current version is 2.
+name: String, required
+type: String, required # The type of the component. Must be one of the following: job, data, script, dockerimage, or output.
+version: String, optional # Component version, default is latest.
+contributor: String, optional
+description: String, optional
+
+prerequisites: # Optional
```

Comment:
please specify whether this field is for **job** only or it can also be applied to other types (e.g. **script**)

Reply:
this protocol yaml is job only, specified at the beginning of file